kyo-offheap/native/src/test/scala/kyo/offheap/MemoryTestNative.scala

```diff
@@ -0,0 +1,283 @@
+package kyo.offheap
+
+class MemoryTestNative extends Test:
```

Comment:
There shouldn't be a separate test class for native. The test should be moved to `shared/src` to be reused for both JVM and Native.